### DISCUSSION TITLE
Apache CouchDB

### DISCUSSION THREAD

**Discussion by sn0opy** [selfhst/icons#7] [New Icon Request]
https://couchdb.apache.org/

SVG logos incl. source files: https://github.com/apache/couchdb-www/tree/asf-site/CouchDB-visual-identity/logo
Also available on Wikipedia: https://commons.wikimedia.org/wiki/File:Apache_CouchDB_logo.svg

**Comment by shollyethan** [comment]
CouchDB has been added!
